GLOSSARY.md

```diff
@@ -0,0 +1,22 @@
+# Cog Glossary
+
+## Bundle
+## Command
+### Subcommand
+## Group
+## Permission
+  * A way to define what commands can be accessed
+  * Come with bundles
+  * Can be created custom in the `site:` namespace
+
+### Examples:
+  * `operable:manage_commands`
+  * `ec2:delete`
+
+## Relay
+## Role
+  * Group of permissions
+  * Can be attached to a group
+## Rule
+## Tag
+## User
```

Comment:
User - people executing Cog commands. Currently, permissions must be attached to a Cog user in some manner(whether directly or via groups or roles). A Cog user can have multiple accounts in different chat adapters that are associated with the Cog user.  https://github.com/operable/cog/wiki/A-Tour-Through-Cog%27s-Authorization-System#users (This definition may change slightly if/when we implement Issue #229)